config-templates/redhat/odf-remote/4.10/custom-parameters.json

```diff
@@ -4,21 +4,27 @@
     "displayname": "OSD pod volume size",
     "name": "osd-size",
     "default": "100Gi",
+    "regex": "^[1-9]{1}[0-9]{0,3}(?:Gi|Ti)$",
```

Comment:
@akgunjal We are allowing max value till 1000Ti as per this validation, will try to check if we have some doc on redhat side for this otherwise should be fine for now...